docs/src/userguide/debugging.rst

```diff
@@ -21,6 +21,8 @@ source, and then running::
     make
     sudo make install
 
+Installing the Cython debugger can be quite tricky. `This installation script and example code <https://gitlab.com/volkerweissmann/cygdb_installation>`_ might be useful.
```

Comment:
Are you really asking users to build their own gdb package here? After your changes, the system supplied one should just work, no?

Reply:
Interesting question. Even before this PR, we asked users to build their own gdb package using python2 (see lines 20-22 in debugging.rst).
I just tested it with the newest Cython version with my system supplied gdb that uses python3 and it works in dfab326da347a89a8941e9b1247aaa130e0e7ef0 . Do you know why we once told the user to use python2? If not, we can remove this part.